aws/resource_aws_db_instance.go

```diff
@@ -908,7 +908,13 @@ func resourceAwsDbInstanceCreate(d *schema.ResourceData, meta interface{}) error
 			return nil
 		})
 		if err != nil {
+			if awsErr, ok := err.(awserr.Error); ok {
```

Comment:
Minor nitpick: we have a helper function to simplify this logic (and normally prefer to use SDK provided constants but looks like the [rds package](https://docs.aws.amazon.com/sdk-for-go/api/service/rds/#pkg-constants) doesn't have one here 😉 )

```go
if isAWSErr(err, "InvalidParameterValue", "") {
```